src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -68,6 +68,7 @@
 #include <EnergyPlus/NodeInputManager.hh>
 #include <EnergyPlus/Psychrometrics.hh>
 #include <EnergyPlus/ScheduleManager.hh>
+#include <EnergyPlus/UnitarySystem.hh>
```

Comment:
Why did I need this?

Reply:
The factory is still a member of UnitarySystem so unless the factory is moved to DataHVACSystems this is needed.